### ISSUE TITLE
Delete game icon to small and not clickable.

### ISSUE THREAD

**Issue by Ud71p** [btouellette/concarneau#7]
Recently (for the last 3 weeks or so) the mechanism of removing games seems to be broken. Previously, when I clicked on the 'edit' button, an icon appeared on every game in the game list allowing to delete a game. Now, I can click on 'edit', but instead of an icon, a very small thing appears on every game (maybe it's a comma, maybe a partial question mark), but I'm unable to click on it to remove a game.

As a workaround, one can locate the closing icon in dev tools inspector, and execute .click() on it from the console.

As a side effect I've noticed that with many (around 10) games open, the performance deteriorates, and the game sometimes becomes unstable (the move is not executed, I need to refresh, and sometimes I'm getting 'application error'). But maybe this performance and stability has nothing to do with number of open games, maybe it's just a coincidence.

**Comment by btouellette** [comment]
Did the icon to start a new game show up for you with my previous update? The edit/delete/add icons are all from the same icon set which wasn't correctly referenced but it should be fixed now.

Edit/delete is working normally for me. A screenshot of the broken icon and the full text of the "menu-game-listing" div might help me narrow down what is going on in your case.

On the network tab of the dev tools are there any non-200/101 status codes? And are there any errors in the dev tools console? Either when you're trying to delete a game or when you're having performance issues.

Performance shouldn't change really at all based on how many games you have going since it only renders one of them at a time. I've got 30 open as a test now and don't notice any slowdown.

**Comment by Ud71p** [comment]
Yes, the icon to start a new game did show up for you with your previous update. So did the edit icon, but not delete.

Performance issue: please disregard it, I guess the cloud was slow, I'll file a separate issue if this ever reoccurs.

I have solved the mystery of the delete icon. The problem occurs for me because I have disabled foreign (website's) fonts in my browser. E.g. in Mozilla-based browsers it's the browser.display.use_document_fonts in about:config, also accessible from normal settings.

It might be possible to reproduce my problem by disabling foreign fonts.

If I enable document fonts, I can see the delete icon. But I'm not going to do it permanently (see below why).

I have found a work-around. I added the following CSS rule to my custom style:
.fa-trash-o::before {content: 'X';}

The standard content (without my work-around) is U+F014, a character in unicode's private use block.

The reason why I disabled document fonts is security. Carefully crafted font can be used to hack my machine:

https://soylentnews.org/article.pl?sid=15/07/23/1255221

But I know other people who have done the same due to aesthetics, usability (semi-blind people like their own fonts), and in the older days some even did it just to speed-up browsing.

I know many web-apps use font icons and I know there is a new holy war emerging between font icons and images. Personally I prefer images, as I disabled fonts, but I could tolerate font icons if they were defined on some other, common characters (e.g. 'X' for delete icon, '+' for add icon, etc.) which would serve as a fallback for those who disable document fonts.

So to summarize, it's up to you whether you want the game to support people who disable document fonts. My work-around works for me, so I don't care so much, but maybe there are other players who suffer if they don't apply the work-around (my friend who I often play with is one such example).

**Comment by btouellette** [comment]
Interesing, Wasn't aware of that. It's odd that just the trash icon isn't showing up since the Edit and Add game buttons are also from the same font. Maybe there is some way I can have it fall back on standard X/+/- characters in the event that the font isn't loaded. Thanks for all the info.

**Comment by btouellette** [comment]
I found SVG exports of all the FA icons (https://github.com/encharm/Font-Awesome-SVG-PNG) and migrated all font icon usage to SVG icons: https://github.com/btouellette/concarneau/commit/bfff55ed74d9ea753d875dcd1e77a622d1a161de

Let me know if anything looks the wrong size or any functionality was broken for you. Non-trivial change but pretty sure I got everything ported over properly.
